File: Lib/Telerik/Examples/QuickStart/ChartView/ChartTypes/Scatter/HourlyEarnings.vb
Imports System.Text

Namespace Telerik.Examples.WinControls.ChartView.ChartTypes.Scatter
	Public Class HourlyEarnings
'INSTANT VB NOTE: The variable sector was renamed since Visual Basic does not allow class members with the same name:
		Private sector_Renamed As String
'INSTANT VB NOTE: The variable wage was renamed since Visual Basic does not allow class members with the same name:
		Private wage_Renamed As Double
'INSTANT VB NOTE: The variable age was renamed since Visual Basic does not allow class members with the same name:
		Private age_Renamed As Integer

		Public Sub New(ByVal sector As String, ByVal wage As Double, ByVal age As Integer)
			Me.sector_Renamed = sector
			Me.wage_Renamed = wage
			Me.age_Renamed = age
		End Sub

		Public ReadOnly Property Sector() As String
			Get
				Return Me.sector_Renamed
			End Get
		End Property

		Public ReadOnly Property Wage() As Double
			Get
				Return Me.wage_Renamed
			End Get
		End Property

		Public ReadOnly Property Age() As Integer
			Get
				Return Me.age_Renamed
			End Get
		End Property
	End Class
End Namespace

File: Lib/Telerik/Examples/QuickStart/ChartView/ChartTypes/Scatter/HourlyEarnings.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Telerik.Examples.WinControls.ChartView.ChartTypes.Scatter
{
    public class HourlyEarnings
    {
        private string sector;
        private double wage;
        private int age;

        public HourlyEarnings(string sector, double wage, int age)
        {
            this.sector = sector;
            this.wage = wage;
            this.age = age;
        }

        public string Sector
        {
            get { return this.sector; }
        }

        public double Wage
        {
            get { return this.wage; }
        }

        public int Age
        {
            get { return this.age; }
        }
    }
}
